### ISSUE TITLE
[BUG] Race harness issues

### ISSUE THREAD

**Issue by Mambaa19** [ParadoxGamingRP/bugs-and-suggestions#1202]
### Your Discord Name

Mambaa_19#4871

### A clear and concise description of what the bug is

So when i go to put on my race harness before a race as soon as the bar finishes putting it on it'll just bring the bar up that shows the harness being taken off so as soon as i put the harness on it comes right off in simple terms and it also happend where i had it on and i couldnt take it off as as soon as i tried taking it off it put it right back on again.

### Steps to Reproduce the Bug

Simply put on the harness or trying to take off the harness it will do the opposite of what you're trying to do.

### Expected Behavior

Hopefully a fix to make it so the harness goes on and off as we please

### Screenshots/Videos

https://medal.tv/games/gta-v/clips/XiMHQXeNbCuY3/d1337eelmFPD?invite=cr-MSx0MTYsMTAzMDY5Nzk5LA
